core/services/fluxmonitor/flux_monitor_simulated_blockchain_test.go

```diff
@@ -569,3 +571,40 @@ func TestFluxMonitor_HibernationMode(t *testing.T) {
 	case <-time.After(5 * time.Second):
 	}
 }
+
+func TestFluxMonitor_LatestRoundData(t *testing.T) {
+	key := cltest.MustGenerateRandomKey(t)
+	fa := setupFluxAggregatorUniverse(t, key)
```

Comment:
This setup is kind of expensive right - could we just work this into another test as a subtest?

Reply:
@connorwstein This test should get removed once https://www.pivotaltracker.com/story/show/176383726 is addressed. Mind if we just leave it in it's own test for now? Also the other two simulated backend tests are already fairly lengthy, so I'd prefer not to add additional complexity if we can avoid it.